src/main/webapp/style.css

```diff
@@ -103,6 +103,10 @@ a:hover {
   min-height: 20%;
 }
 
+.step-img-4 {
+  background-image: url(images/graduationbackground.png);
```

Comment:
The background image does not fit in super well with the the current pixel art/ retro gaming theme. Maybe design the hats with pixel art? 

I love the rest of the aesthetic choices though, especially the outfits!

Reply:
I will either make a background image all of pixels or try and find one

Reply:
+1